### ISSUE TITLE
Unhandled promise rejection while loading views hide the actual error

### ISSUE THREAD

**Issue by gabor-farkas** [aurelia/templating#144]
Exceptions like this:
```
Unhandled promise rejection Error: Error loading "app.html!view" at http://localhost:9000/dist/app.html(…)(anonymous function)
@ shim.min.js:1444b.exports
@ shim.min.js:453b.(anonymous function)
@ shim.min.js:1625f
@ shim.min.js:1596q
@ shim.min.js:1600
```
Can easily hide actual error messages.
(I have place a breakpoint in shim.js to find the particular exception, which is for my current case is a strange thing with chrome:
"Error: Error loading "app.html!view" at http://localhost:9000/dist/app.html↵There was no template element found in 'http://localhost:9000/dist/app.html'.↵    at DefaultLoader.findTemplate (http://localhost:9000/jspm_packages/github/aurelia/loader@0.8.3/aurelia-loader.js:200:15)↵    at http://localhost:9000/jspm_packages/github/aurelia/loader@0.8.3/aurelia-loader.js:188:22↵    at f (http://localhost:9000/jspm_packages/npm/core-js@0.9.18/client/shim.min.js:1415:56)↵    at http://localhost:9000/jspm_packages/npm/core-js@0.9.18/client/shim.min.js:1423:13↵    at b.exports (http://localhost:9000/jspm_packages/npm/core-js@0.9.18/client/shim.min.js:453:24)↵    at b.(anonymous function) (http://localhost:9000/jspm_packages/npm/core-js@0.9.18/client/shim.min.js:1625:11)↵    at Number.f (http://localhost:9000/jspm_packages/npm/core-js@0.9.18/client/shim.min.js:1596:24)↵    at q (http://localhost:9000/jspm_packages/npm/core-js@0.9.18/client/shim.min.js:1600:11)"
)

**Comment by sabau** [comment]
Same problem with firefox, after a fresh install by yo, I was looking for 
http://localhost:9000/

    GET XHR http://localhost:9000/dist/main [HTTP/1.1 404 Not Found 2ms]

    "Unhandled promise rejection" "Error loading "main" at http://localhost:9000/dist/main
    Not Found: http://localhost:9000/dist/main" shim.min.js:1444:98
    i/</<() shim.min.js:1444
    b.exports() shim.min.js:453
    <anonymous> shim.min.js:1625
    f() shim.min.js:1596
    q() shim.min.js:1600

**Comment by EisenbergEffect** [comment]
What's a good way to reproduce this so I can work on fixing the error output?

**Comment by gabor-farkas** [comment]
Sorry I didn't have much time lately.
I had the problem above with loader 0.8.3, but I can still reproduce it with 0.8.7. If I just pick the project I created back then with the yeoman generator, change app.html to something that doesn't have a root template element, and then run 'gulp watch', it has this error.
I now cannot reproduce the same thing with the typescript starter kit for 1.0.0-beta, so the issue must have been solved since then. If I do the same thing to app.html, the error is properly reported on the browser console.
(I guess the yeoman generator is no longer supported, as the changed docs page doesn't mention it).
